elephant/statistics.py

```diff
@@ -524,3 +525,101 @@ def instantaneous_rate(spiketrain, sampling_period, form, sigma, m_idx=None,
                                  units=pq.Hz, t_start=t_start)
 
     return rate
+
+
+def psth(spiketrains, binsize, t_start=None, t_stop=None, output='counts',
+         clip=False):
```

Comment:
In the discussion of the binned spike train class, we agreed to drop the name "clip", as it is confusing/ambiguous for some people, and use a name involving "bool" or "binary" instead.